docs/api/app.md

```diff
@@ -725,10 +725,16 @@ Returns `string` - User operating system's locale two-letter [ISO 3166](https://
 
 ### `app.getSystemLocale()`
 
-Returns `string` - The current system locale. On Windows and Linux, it is fetched using Chromium's `i18n` library. On macOS, the `NSLocale` object is used instead.
+Returns `string` - The current system locale. On Windows and Linux, it is fetched using Chromium's `i18n` library. On macOS, `[NSLocale currentLocale]` is used instead.
+
+Note that the system locale is not necessarily the preferred system language. For example, The system locale on Windows 11 returns the locale used for the region. On macOS Monterey, it returns a string containing a mix of the language and the region. For example, setting the most preferred language to Brazilian Portuguese but setting the region to Finland returns `pt-FI` instead of `pt-BR`.
```

Comment:
There are 2 "For example" here, putting the examples in a list might make them easier to read.

Also can you add a paragraph on when to use `getSystemLocale` or `getPreferredSystemLanguages`?